index.html

```diff
@@ -49,5 +49,165 @@ <h1 class="headline-title">Hey there. I'm Rachel</h1>
             <button type="button"><img src="./img/Union.png"></button>
         </div>
     </section>
-    </body>
-    </html>
\ No newline at end of file
+    <!--WORKS SECTION-->
+    <section>
+        <div class="work">
+            <h1>My Recent Works</h1>
+            <div class="indicator-image">
+            <img src="./img/Indicator.svg" alt="">
+            </div>
+            <article>
+                <div class="works-card">
+                    <div class="works-card-image">
+                        <img src="./img/Img Plaholder.svg" alt="">
+                    </div>
+                    <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="works-card-skills">
+                        <li>ruby on rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                        <li>html</li>
+                    </ul>
+                    <button type="button" class="project-btn">See Project</button>
+                </div>
+            </article>
```

Comment:
- [ ] Consider sticking to the Figma design
- your design
![image](https://user-images.githubusercontent.com/52143815/192009794-702820a7-578d-4f31-9fb9-3275fa37225d.png)

- figma design
![image](https://user-images.githubusercontent.com/52143815/192009927-8a62c46e-27ee-4835-be1e-6cef0303f1bf.png)